R/module_init_data.R

```diff
@@ -56,7 +56,7 @@ ui_init_data <- function(id, data) {
 #' @rdname module_init_data
 srv_init_data <- function(id, data, modules, filter = teal_slices()) {
   checkmate::assert_character(id, max.len = 1, any.missing = FALSE)
-  checkmate::assert_multi_class(data, c("teal_data", "teal_data_module", "reactive", "reactiveVal"))
+  checkmate::assert_multi_class(data, c("teal_data", "teal_data_module", "reactive", "reactiveVal", "qenv.error"))
```

Comment:
I think this is initialization failure and app-developer shouldn't be able to start the application. If somebody want to start an app without data and maybe add something in `teal_transform_module` then `data = teal_data()`. I don't think we should enable passing `qenv.error`
```suggestion
  checkmate::assert_multi_class(data, c("teal_data", "teal_data_module", "reactive", "reactiveVal"))
```